specification/figures/xss-example-human-text.md

```diff
@@ -1,7 +1,7 @@
 # Human readable text based on XSS-Example.pdf
```

Comment:
Since there is no longer a PDF, this heading should be changed.

Reply:
clerical oversight, resolved